modules/administration-guide/partials/proc_migration-to-devworkspace-engine.adoc

```diff
@@ -0,0 +1,208 @@
+  USER_ID=${IDS[0]}
+  OPENSHIFT_USER_ID=${IDS[1]}
+
+  sed -i -e "s|${USER_ID}|${OPENSHIFT_USER_ID}|g" "${DUMP}"
+
+  echo "[INFO] Migrated User ID from \"${USER_ID}\" to \"${OPENSHIFT_USER_ID}\""
+done < "${ALL_USERS}"
+----
+
+include::partial$snip_restoring-che-database.adoc[]
+
+. Add empty users profiles:
++
+[subs="+quotes,+attributes"]
+----
+ALL_USERS={prod-id}-users.txt
+while IFS= read -r line
+do
+  IDS=($line)
+  OPENSHIFT_USER_ID=${IDS[1]}
+
+  {orch-cli} exec $POSTGRES_POD -n {prod-namespace}  -- bash  -c "psql ${CHE_POSTGRES_DB} -tAc \"insert into profile(userid) values ('${OPENSHIFT_USER_ID}');\""
+
+  echo "[INFO] Added profile for \"${OPENSHIFT_USER_ID}\""
+done < "${ALL_USERS}"
+----
+
+. Find Cluster Service Version name:
++
+[subs="+quotes,+attributes"]
+----
+CSV=$({orch-cli} get subscription {prod-id} -n {prod-namespace} -o jsonpath="{.status.currentCSV}")
+----
+
+. Delete Cluster Service Version:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} delete csv ${CSV} -n {prod-namespace}
+----
+
+. Delete Subscription:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} delete subscription {prod-id} -n {prod-namespace}
+----
+
+. Enable {devworkspace} engine:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
+'[{"op": "replace", "path": "/spec/devWorkspace/enable", "value": true}]'
+----
+
+. Enable single-host exposure strategy:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
+'[{"op": "replace", "path": "/spec/server/serverExposureStrategy", "value": "single-host"}]'
+----
+
+. Delete {identity-provider} route:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} delete route keycloak -n {prod-namespace}
+----
+
+. Delete {identity-provider} service:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} delete service keycloak -n {prod-namespace}
+----
+
+. Delete {identity-provider} deployment:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} delete deployment keycloak -n {prod-namespace}
+----
+
+. Create a new subscription to `{prod-tech-preview-channel}` channel:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: {prod-id}
+  namespace: openshift-operators
+spec:
+  channel: {prod-tech-preview-channel}
+  installPlanApproval: Automatic
+  name: {prod-tech-preview-olm-package}
+  source: {prod-catalog-source}
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+. Wait until Operator is ready:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} wait --for=condition=ready pod -l app.kubernetes.io/component={prod-operator} -n openshift-operators --timeout=120s
+----
+
+. Wait until {prod-short} is ready:
++
+[subs="+quotes,+attributes"]
+----
+{orch-cli} wait --for=condition=ready pod -l app.kubernetes.io/component={prod-deployment} -n {prod-namespace} --timeout=240s
+----
```

Comment:
I'd recommend to add a step how to open login page

Reply:
Also, I had an issue with login in the same tab after the update procedure (I left a browser window opened with Che dashboard and then just refreshed the page and tried to login). When I closed the private window and open the route again in a new one - it worked.

Reply:
added